source/sass/views/regrets-reporter-landing-page.scss

```diff
@@ -47,4 +47,78 @@
     width: 100%;
     height: 100%;
   }
+
+  .share-button-container {
```

Comment:
Hmm I feel like you could have probably do a majority of this with tailwind classes including the background images as well instead of writing a sass file. I do not think it is required for the ticket but I think it would be a great refactor task in the future. Can totally walk through this with you as well!

Reply:
Would love to Simon! Going to take a note to make a new ticket to walk through this!